spec/rubocop/cop/rails/migration_file_name_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::MigrationFileName, :config do
+  let(:filename) { 'db/migrate/20220101050505_create_users.rb' }
+
+  context 'when the file name matches its class' do
+    it do
+      expect_no_offenses(<<~RUBY, filename)
+        class CreateUsers < ActiveRecord::Migration[7.0]
+        end
+      RUBY
+    end
+  end
+
+  context 'when the file name does not match its class' do
+    it do
+      expect_offense(<<~RUBY, filename)
+        class SellBooks < ActiveRecord::Migration[7.0]
+              ^^^^^^^^^ The class name does not match with the file name.
```

Comment:
It would be user-friendly to indicate the class name to replace.

```suggestion
              ^^^^^^^^^ Replace with `CreateUsers` that matches the file name.
```

Reply:
And I think this cop can provide autocorrect the class name.